contract-ui/tabs/overview/components/MarketplaceDetails.tsx

```diff
@@ -0,0 +1,366 @@
+import {
+  ListingStats,
+  ListingStatsV3,
+} from "../../listings/components/listing-stats";
+import {
+  AspectRatio,
+  Flex,
+  GridItem,
+  SimpleGrid,
+  Skeleton,
+  SkeletonText,
+  useBreakpointValue,
+} from "@chakra-ui/react";
+import {
+  useContract,
+  useDirectListings,
+  useDirectListingsCount,
+  useEnglishAuctions,
+  useEnglishAuctionsCount,
+  useListings,
+  useListingsCount,
+} from "@thirdweb-dev/react";
+import {
+  AuctionListing,
+  DirectListing,
+  DirectListingV3,
+  EnglishAuction,
+  ListingType,
+  Marketplace,
+  MarketplaceV3,
+} from "@thirdweb-dev/sdk";
+import { useTabHref } from "contract-ui/utils";
+import { BigNumber } from "ethers";
+import { useMemo } from "react";
+import {
+  Badge,
+  Card,
+  Heading,
+  Text,
+  TrackedLink,
+  TrackedLinkProps,
+} from "tw-components";
+import { AddressCopyButton } from "tw-components/AddressCopyButton";
+import { NFTMediaWithEmptyState } from "tw-components/nft-media";
+
+type Listing = DirectListing | AuctionListing;
+type ListingV3 = DirectListingV3 | EnglishAuction;
+
+type ListingData = {
+  asset: Listing["asset"] | ListingV3["asset"];
+  type: "direct-listing" | "english-auction";
+  sellerAddress: Listing["sellerAddress"] | ListingV3["creatorAddress"];
+  currencyValue:
+    | Listing["buyoutCurrencyValuePerToken"]
+    | DirectListingV3["currencyValuePerToken"]
+    | EnglishAuction["buyoutCurrencyValue"];
+};
+
+type MarketplaceDetailsProps = {
+  contractAddress: string;
+  contractType: "marketplace" | "marketplace-v3";
+  trackingCategory: TrackedLinkProps["category"];
+};
+
+interface MarketplaceDetailsVersionProps<T> {
+  contract: T;
+  trackingCategory: TrackedLinkProps["category"];
+}
+
+export const MarketplaceDetails: React.FC<MarketplaceDetailsProps> = ({
+  contractAddress,
+  contractType,
+  trackingCategory,
+}) => {
+  const { contract } = useContract(contractAddress, contractType);
+
+  if (contractType === "marketplace" && contract) {
+    return (
+      <MarketplaceV1Details
+        contract={contract as Marketplace}
+        trackingCategory={trackingCategory}
+      />
+    );
+  } else if (contractType === "marketplace-v3" && contract) {
```

Comment:
never check for `contractType === "marketplace-v3"`, you need to detect if either direct listings or english auctions extension exist, then you show this.